PFR/MoreRuzsaDist.lean

```diff
@@ -82,10 +82,36 @@ variable {Ω Ω' Ω'' Ω''' G T : Type*}
 
 variable {X : Ω → G} {Y : Ω' → G} {Z : Ω'' → G} [FiniteRange X] [FiniteRange Y] [FiniteRange Z]
 
+-- Mathlib PR #12918 has been merged.
+@[to_additive]
+lemma ProbabilityTheory.IdentDistrib.inv {α : Type*} {β : Type*} {γ : Type*}
+    [MeasurableSpace α] [MeasurableSpace β] [MeasurableSpace γ] {μ : Measure α} {ν : Measure β}
+    {f : α → γ} {g : β → γ} [Inv γ] [MeasurableInv γ] (h : IdentDistrib f g μ ν) :
+    IdentDistrib f⁻¹ g⁻¹ μ ν := h.comp measurable_inv
```

Comment:
You should merge master into your branch to remove the lemma

Reply:
Ok @YaelDillies , we'll do it in a few minutes.

Reply:
I've also noticed that the checks for this draft PR are failing too.

Reply:
Uh that's because `says` is a pain. I'll fix it tomorrow

Reply:
Ok, thanks.